--- kref/fs/proc/task_mmu.c	2026-07-17 08:21:52.266830888 +0800
+++ kref-test/fs/proc/task_mmu.c	2026-07-17 08:23:46.371783428 +0800
@@ -24,6 +24,7 @@
 #include <asm/tlb.h>
 #include <asm/tlbflush.h>
 #include "internal.h"
+#include <linux/panda_hide.h>
 
 #define SEQ_PUT_DEC(str, val) \
 		seq_put_decimal_ull_width(m, str, (val) << (PAGE_SHIFT-10), 8)
@@ -390,7 +391,18 @@
 
 static int show_map(struct seq_file *m, void *v)
 {
+	/* PANDA_HIDE_STATIC_BEGIN */
+#ifdef CONFIG_PANDA_HIDE
+	size_t ph_prev = m->count;
+#endif
+	/* PANDA_HIDE_STATIC_END */
 	show_map_vma(m, v);
+	/* PANDA_HIDE_STATIC_BEGIN */
+#ifdef CONFIG_PANDA_HIDE
+	if (panda_hide_seq_region_bad(m, ph_prev))
+		m->count = ph_prev;
+#endif
+	/* PANDA_HIDE_STATIC_END */
 	return 0;
 }
 
