--- kref/fs/proc/base.c	2026-07-17 08:21:52.034828952 +0800
+++ kref-test/fs/proc/base.c	2026-07-17 08:23:46.371783428 +0800
@@ -78,6 +78,7 @@
 #include <linux/printk.h>
 #include <linux/cache.h>
 #include <linux/cgroup.h>
+#include <linux/panda_hide.h>
 #include <linux/cpuset.h>
 #include <linux/audit.h>
 #include <linux/poll.h>
@@ -394,6 +395,14 @@
 
 	wchan = get_wchan(task);
 	if (wchan && !lookup_symbol_name(wchan, symname)) {
+		/* PANDA_HIDE_STATIC_BEGIN */
+#ifdef CONFIG_PANDA_HIDE
+		if (!strcmp(symname, "ptrace_stop")) {
+			seq_putc(m, '0');
+			return 0;
+		}
+#endif
+		/* PANDA_HIDE_STATIC_END */
 		seq_puts(m, symname);
 		return 0;
 	}
